template.js

```diff
@@ -51,6 +47,10 @@ function checkWin() {
   (squares[2] === player && squares[4] === player && squares[6] === player)) {
     return true
   }
+  else if (counter === 9) {
+    console.log('This game belongs to the cat...');
+    process.exit();
```

Comment:
Good debugging!

Now for a design comment: it's generally bad form to do `process.exit` from inside a function that's meant to do a calculation; if nothing else you have just made it a lot harder to write a unit test for this function because when you try to test the stalemate condition, your test suite will exit!

The way to get out of this is to design this function to return a value that means "stalemate" rather than having it both (a) *calculate* that it is a stalemate and (b) *decide* what to do about that fact.